theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -155,12 +155,13 @@ def build_deb_package_steps(packages_to_build, version, repoowner = 'theforeman'
             echo "scratch build: uploading to stagingdeb/${suite}/${component}"
         }
 
-        if (!repos.containsKey("${suite}-${component}")) {
+        if (!repos["${suite}-${component}"]) {
             repos["${suite}-${component}"] = [suite: suite, component: component, deb_paths: [], type: release_type]
         }
         repos["${suite}-${component}"].deb_paths.add("${deb_path}/*deb")
     }
 
+    echo "Repos to rsync: ${repos}"
```

Comment:
I don't see this in the jenkins job you linked. Am I missing it?

Reply:
This part was added afterwards just to help debugging in the future.